pylmod/tests/test_gradebook.py

```diff
@@ -110,6 +110,195 @@ class TestGradebook(BaseTest):
         ]
     }
 
+    OPTIONS_BODY = {
+        'data':
+        {
+            'accessLevel': 'class',
+            'archived': False,
+            'calc_on_approved_only': False,
+            'configured': None,
+            'courseName': '',
+            'courseNumber': 'mitxdemosite',
+            'deriveOverallGrades': False,
+            'gradebookEwsEnabled': False,
+            'gradebookId': 1293808,
+            'gradebookName': 'Gradebook for mitxdemosite',
+            'gradebookReadOnly': False,
+            'gradebookVisibleToAdvisors': False,
+            'graders_change_approved': False,
+            'hideExcuseButtonInUI': False,
+            'homeworkBetaEnabled': False,
+            'membershipQualifier': '/project/mitxdemosite',
+            'membershipSource': 'stellar',
+            'student_sees_actual_grades': True,
+            'student_sees_category_info': True,
+            'student_sees_comments': True,
+            'student_sees_cumulative_score': True,
+            'student_sees_histograms': True,
+            'student_sees_submissions': False,
+            'ta_approves': False,
+            'ta_change_approved': False,
+            'ta_configures': False,
+            'ta_edits': False,
+            'use_grade_weighting': False,
+            'usingAttendance': False,
+            'versionCompatible': 4,
+            'versionCompatibleString': 'General Availability'
+        },
+    }
+
+    STAFF_BODY = {
```

Comment:
Wouldn't this be adequate as just two users in two different roles?  I don't see a test for checking that each role type is in the return data.  Generally, I'd prefer to only include test data that is actually tested.  That said, if you want to add tests so that all of this data is used, that would be ideal.